src/frontend/src/pages/PIAIntakeForm/index.tsx

```diff
@@ -515,6 +532,28 @@ const PIAFormPage = () => {
                 </div>
               </div>
             )}
+            {!pathname.includes('intake') && (
+              <div>
```

Comment:
Can we add this to a common component? This will also be used in view mode.

Reply:
done